src/styles/_card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 202px;
+  padding: 32px 16px 16px;
+  background-color: $fontColor__white;
+  border: 1px solid $borderColor_light;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin: 40px 0 0;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $fontColor__darkblue;
+  }
+
+  &__id-good {
+    margin-top: 4px;
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $fontColor__lightgray;
+  }
+
+  &__price {
+    display: flex;
+    margin-top: 24px;
+    justify-content: space-between;
+    align-items: center;
+
+    &-title {
+      font-size: 12px;
+      line-height: 14px;
+      font-weight: 400;
+      color: $fontColor__lightgray;
+    }
+
+    &-text {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+      color: $fontColor__darkblue;
+    }
+  }
+
+  &__button {
+    text-align: center;
+    display: block;
+    margin-top: 16px;
+    box-sizing: border-box;
```

Comment:
Since you define `box-sizing: border-box;` inside `*` selector you can remove it from other blocks

Reply:
Oh... I have forgotten to remove this property... Thank you very much!